apps/tup-ui/src/pages/Projects/Projects.module.css

```diff
@@ -1,7 +1,5 @@
 .project-header {
     font-size: 2.6rem;
-    margin-top: 12px;
-    margin-left: 24px;
```

Comment:
**Explanation:** I moved spacing to `PageLayout`, and design dropped the weird nav link offset that required this `margin-left` solution.